plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/apsadmin/portal/PageActionAspect.java

```diff
@@ -256,15 +198,28 @@ protected IPage buildPageMetadataForUpdate(HttpServletRequest request, PageActio
         IPage seoPage = null;
         String pagecode = action.getPageCode();
         ApsProperties descriptions = new ApsProperties();
+        ApsProperties langKeywordsKey = new ApsProperties();
         String friendlyCode = request.getParameter(PARAM_FRIENDLY_CODE);
-        String xmlConfig = request.getParameter(PARAM_XML_CONFIG);
         Iterator<Lang> langsIter = this.getLangManager().getLangs().iterator();
         while (langsIter.hasNext()) {
             Lang lang = (Lang) langsIter.next();
             String titleKey = PARAM_DESCRIPTION_PREFIX + lang.getCode();
             String title = request.getParameter(titleKey);
             if (null != title) {
-                descriptions.put(lang.getCode(), title.trim());
+                PageMetatag meta = new PageMetatag(lang.getCode(), "description", title.trim());
```

Comment:
Does this have test coverage

Reply:
In test class TestPageAction (in test package)